cloudformation_templates/app_base.j2

```diff
@@ -281,6 +282,74 @@
       }
     },
 
+    {{ metrics.metric_filter("HTTP5xxMetricFilter",
+                             pattern="$.logType = apache-access && $.status = 5*",
+                             metric_name="HTTP5xx",
+                             metric_value=1) }},
+
+    {{ metrics.metric_filter("HTTPNon5xxMetricFilter",
+                             pattern="$.logType = apache-access && $.status != 5*",
+                             metric_name="HTTP5xx",
+                             metric_value=0) }},
+
+    {{ metrics.alarm("HTTP5xxCountAlarm",
+                    metric_name="HTTP5xx",
+                    statistic="Sum",
+                    period=60,
+                    evaluation_periods=1,
+                    threshold=10,
```

Comment:
Agreed, I copied over the original threshold with the idea that I'd reduce them in a separate pull request which wouldn't block this one if there was disagreement. I think two people is enough though.